assets/style.css

```diff
@@ -0,0 +1,7 @@
+* {
+  color: blue;
+}
+
+h1 {
+  color: blueviolet;
+}
```

Comment:
- [ ] It is a good practice to use class selector for the styling of the page content than element selector. This prevents unintended conflicts.